Source/JavaScriptCore/b3/B3SIMDValue.h

```diff
@@ -25,6 +25,7 @@
 
 #pragma once
 
+#include "b3/B3Opcode.h"
```

Comment:
nit:  add -header-insertion=never to your server arguments to stop it from inserting silly headers all the time